cargo-lock/src/package/source.rs

```diff
@@ -130,7 +138,7 @@ impl SourceId {
         Self::new(SourceKind::Git(reference), url.clone())
     }
 
-    /// Creates a SourceId from a registry URL.
+    /// Creates a SourceId from a (git) registry URL.
```

Comment:
What's up with this?

Reply:
Oh, that was a hangover from my first patch. Have synced with the cargo source now.